fix class's typo (#1330)
exercism/ruby: concepts/modules/introduction.md

Some times you don't need the overhead of creating an object with state.
In these cases, a `module` can be used.

> A module is very similar to a class (in fact, `Module` is `Class`'s parent in the object hierarchy) - the main difference being that rather than using instance methods, we use class methods.

Reviewer: ```suggestion
A module is very similar to a class (in fact, `Module` an ancestor to `Class` in the object hierarchy) - 
```
```suggestion
A module is very similar to a class (in fact, `Module` is an ancestor of `Class` in the object hierarchy) - the main difference being that rather than using instance methods, we use class methods.
```

Reviewer: I think this simplifies the language as well as corrects the spelling.
